libretro/LibretroGraphicsContext.h

```diff
@@ -71,13 +71,13 @@ class LibretroD3D9Context : public LibretroHWRenderContext {
 	public:
 	LibretroD3D9Context() : LibretroHWRenderContext(RETRO_HW_CONTEXT_DIRECT3D, 9) {}
 	bool Init() override { return false; }
-#if 0
-   void InitDrawContext() override
-   {
-      draw_ = Draw::T3DCreateDX9Context();
-      draw_->CreatePresets();
-   }
-#endif
+
```

Comment:
Here it was #if 0,  but you enabled it. Why wasn't it needed before or why is it needed now?

Reply:
that part is just placeholder/deadcode at the moment since there is no D3D support in libretro yet. the main reason it was initially disabled was because the build was missing the thin3d files, but that it isn't the case anymore.